### DISCUSSION TITLE
Cannot uninstall

### DISCUSSION THREAD

**Discussion by Shewbe** [tom-james-watson/breaktimer-app#182] [Q&A]
I installed on macOS 11.6.1 and I cannot uninstall, anyone know why? I also have no way of changing the settings, there is no UI?

**Comment by InSasquatchCountry** [comment]
The app should appear in you top menu bar. As for uninstalling, you’re talkin’ you can’t delete the breaktimer.app? Maybe try https://freemacsoft.net/appcleaner/

**Comment by tom-james-watson** [comment]
Everytime somebody has mentioned not being able to uninstall the app, it has been because they also had the old chrome extension installed. Is that maybe the case?

**Reply by tom-james-watson** [reply]
If so, obviously you'll want to remove the extension.
